packages/@react-aria/selection/src/useTypeSelect.ts

```diff
@@ -88,6 +99,8 @@ function getStringForKey(key: string) {
   // See https://www.w3.org/TR/uievents-key/
   if (key.length === 1 || !/^[A-Z]/i.test(key)) {
     return key;
+  } else if (key === 'Spacebar') { // For IE 11
```

Comment:
we don't care about IE 11 anymore if you want to remove this, no sense in dead code :)

Reply:
![bye_ie](https://1.bp.blogspot.com/-oJFa6YjCZ2I/Vo3LeNuRFyI/AAAAAAAACOQ/NHG_jTfv_28/s1600/RIPInternetExplorer-01.png)